pom.xml

```diff
@@ -35,8 +35,23 @@
       <artifactId>appengine-api-1.0-sdk</artifactId>
       <version>1.9.59</version>
     </dependency>
+
+    <dependency>  
```

Comment:
The indent here for your new code seems wrong. there should be 2 extra spaces for things inside the label. 

Can you open a separate PR to fix this?